src/resources/js/download.js

```diff
@@ -169,11 +169,11 @@ $(function() {
 	});
 })
 
-// autoPlatform choooses the platform in the list that best
+// autoPlatform chooses the platform in the list that best
 // matches the user's browser, if it's available.
 function autoPlatform() {
 	// assume 32-bit linux, then change OS and architecture if justified
-	var os = "linux", arch = "386", arm = "";
+	var os = "linux", arch = "amd64", arm = "";
```

Comment:
If we change this default then we need a way to detect 32-bit and set it below.

Reply:
I don't think we need to worry about precision of the detection. It's pretty much best-guess. The User-Agent header is pretty unreliable and some browsers are starting to intentionally give bad values for privacy reasons.

That said I realize that the comment above this line is wrong now, I'll fix that.

Reply:
How do we auto-select 32-bit platforms with this change?

Reply:
From what I remember there isn't even a 32-bit option in the HTML select box

Reply:
Ah, that's true. I removed them with the new site since 386 was our least popular platform.